### PR TITLE
K8s deployment

### PR DESCRIPTION
**Pull request by rkargul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,12 +20,23 @@ Subsequently, run the following commands:
...
 
+Use `minikube` to tunnel the Ingress to `localhost` with:
+
+`minikube tunnel`
+
+For now: In order to be authorized to pull the container registries, you need to have access to a token:
+
+`kubectl create secret docker-registry regcred --docker-server=https://ghcr.io --docker-username=<GITHUB_USERNAME> --docker-password=<TOKEN> --docker-email=<GITHUB_EMAIL>`
+
+The `deployment.yml` contains `imagePullSecrets` with `regred`
```
Comment (Iodine98) [inline README.md]: `regcred`

File: deployment.yml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      - name: app
+        image: ghcr.io/remla23-team06/app:latest
+        ports:
+        - containerPort: 8080
+        env:
+        - name: MODEL_SERVICE_URL
+          valueFrom:
+            configMapKeyRef:
+              name: my-config
+              key: model.service.url
```
Comment (Iodine98) [inline deployment.yml:27]: @rkargul it used to be a secret right? Will it still work with the instructions provided in the README?

**Review by Iodine98 — APPROVED** [review]
If you could resolve the other two comments, feel free to merge it. `deployment.yml` works on my laptop as well.

File: deployment.yml (hunk 2 above)
Comment (rkargul) [reply deployment.yml:27]: @Iodine98 no, it was not a secret, it used to directly provide the URL there, but I added it to a ConfigMap to make use of that functionality. If this is now what you are asking, please let me know.

I believe the Kubernetes deployment Ingress should not be affected by adding ConfigMap to the manifest.

**Pull request merged**